Profiler/Profiler.example.yml

```diff
@@ -1,4 +1,10 @@
+# Whether SSL validation should be globally disabled.
 disableSslValidation: true
+# Upload new trace files every N minutes. Defaults to 5 minutes.
```

Comment:
change to "upload finished trace files"? as new files are locked - only finished ones can be uploaded.